modules/shortcodes/tweet.php

```diff
@@ -111,6 +111,12 @@ static public function jetpack_tweet_url_extra_args( $provider, $url, $args = ar
 		// Twitter doesn't support maxheight so don't send it
 		$provider = remove_query_arg( 'maxheight', $provider );
 
+		// Add Twitter partner ID to track embeds from Jetpack
+		$partner = apply_filters( 'jetpack_twitter_partner_id', 'jetpack' );
```

Comment:
Could you add a docblock to this new filter, so it can be parsed and integrated in [the Jetpack Codex](http://developer.jetpack.com/)? It also helps whoever may want to use the filter later.

[Here is an example of how to format your docblock.](https://github.com/jom/jetpack/blob/9d8957baa281926bb2974e092c0db411c2353ccc/modules/shortcodes/youtube.php#L216-L225)